packages/main/src/TableRowTemplate.tsx

```diff
@@ -0,0 +1,76 @@
+import type TableRow from "./TableRow.js";
+import TableCell from "./TableCell.js";
+import CheckBox from "./CheckBox.js";
+import RadioButton from "./RadioButton.js";
+import Button from "./Button.js";
+
+export default function TableRowTemplate(this: TableRow) {
+	return (
+		<>
+			{ this._hasRowSelector &&
+				<TableCell
+					id="selection-cell"
+					aria-selected={this._isSelected}
+					ui5-table-cell-fixed
+					ui5-table-selection-component
+				>
+					{ this._isMultiSelect ?
+						<CheckBox id="selectionComponent"
+							tabindex={-1}
+							checked={this._isSelected}
+							onChange={this._informSelectionChange}
+							accessible-name={this._i18nRowSelector}
```

Comment:
accessible-name => accessibleName
using properties is recommended as they are type checked (and auto completed)